pages/common/cmark.md

```diff
@@ -0,0 +1,19 @@
+# cmark
+
+> Converts CommonMark Markdown formatted text to HTML
```

Comment:
Since the tool converts to latex also, maybe we should say `.. formatted text to other formats` ?

Reply:
Sounds good to me. :+1:

Reply:
Oh, oops! This mus  have slipped through the net. I'll update this now!